jekyll/_cci2/docker-layer-caching.md

```diff
@@ -17,20 +17,19 @@ Consider reusing the unchanged layers to significantly reduce image build times.
 
 ``` YAML
 - setup_remote_docker:
-    reusable: true    # default - false
-    exclusive: true   # default - true
+    docker_layer_caching: true # default - true  
 ```
 
-## **`reusable`**
+When `docker_layer_caching` is set to `true`, CircleCI will try to reuse your Remote Docker Environment. That is, every layer you built in a previous job will be accessible in the remote environment and CircleCI will attempt to reuse the previous environment when it is possible. However, in some cases your job may run in a clean environment, even if the configuration specifies `docker_layer_caching: true`.
 
-When `reusable` is set to `true`, CircleCI will try to reuse your Remote Docker Environment. That is, every layer you built in a previous job will be accessible in the remote environment and CircleCI will attempt to reuse the previous environment when it is possible. However, in some cases your job may run in a clean environment, even if the configuration specifies `reusable: true`.
+If you run many parallel jobs for the same project that depend on the same environment, all of them will be provided with a Remote Docker Environment. Docker Layer Caching guarantees jobs to have exclusive Remote Docker Environments that other jobs cannot access. However, not all of the jobs will have cached layers, although this behavior is subject to change in a future update.
```

Comment:
"However, not all of the jobs will have cached layers, although this behavior is subject to change in a future update." -> "However, not all of the jobs might have cached layers and not all of the jobs will have identical cache."

Reply:
thanks, changed to "However, all of the jobs may not have cached layers and not all of the jobs will have identical cache."